src/unix/bsd/netbsdlike/openbsd/mod.rs

```diff
@@ -1639,6 +1639,26 @@ pub const SF_ARCHIVED: ::c_uint = 0x00010000;
 pub const SF_IMMUTABLE: ::c_uint = 0x00020000;
 pub const SF_APPEND: ::c_uint = 0x00040000;
 
+// sys/mount.h
+pub const MNT_NOPERM: ::c_int = 0x00000020;
+pub const MNT_WXALLOWED: ::c_int = 0x00000800;
+pub const MNT_EXRDONLY: ::c_int = 0x00000080;
+pub const MNT_EXPORTED: ::c_int = 0x00000100;
+pub const MNT_DEFEXPORTED: ::c_int = 0x00000200;
+pub const MNT_EXPORTANON: ::c_int = 0x00000400;
+pub const MNT_ROOTFS: ::c_int = 0x00004000;
+pub const MNT_NOATIME: ::c_int = 0x00008000;
+pub const MNT_UPDATE: ::c_int = 0x00010000;
+pub const MNT_DELEXPORT: ::c_int = 0x00020000;
+pub const MNT_RELOAD: ::c_int = 0x00040000;
+pub const MNT_FORCE: ::c_int = 0x00080000;
```

Comment:
already defined in `src/unix/bsd/mod.rs`

Reply:
Ahh, sorry.  I tried to filter out stuff like that, but I guess I missed a few.